modules/formulize/templates/admin/ui.html

```diff
@@ -160,6 +160,22 @@
     }
   });
 
+  $(".DCTemplates").click(function(){

```

Comment:
It's not a big deal but "DCTemplates" isn't a great name because I don't know what it means. Something like "delete-templates" would be better.